spec/factories/tags.rb

```diff
@@ -1,5 +1,5 @@
 FactoryBot.define do
-  sequence(:name) { |n| "tag你好#{n}" }
+  sequence(:name) { |n| "tag#{n}" }
```

Comment:
this was useful information but will be a larger change than needs to be reviewed as part of this change.